src/scenes/Jarjestajat/Jarjestaja/components/JulkisetTiedot.js

```diff
@@ -19,31 +19,29 @@ const JulkisetTiedot = props => {
   }
 
   return (
-    <div className="bg-white p-16">
+    <div className="bg-white p-8">
       <div>
-        <div>
-          <Typography>{LUPA_TEKSTIT.PAATOKSET.OTSIKKO.FI}</Typography>
-          <Typography paragraph={true} variant="h6">
-            {LUPA_TEKSTIT.PAATOKSET.VIIMEISIN.FI}
-          </Typography>
-        </div>
+        <Typography><h1>{LUPA_TEKSTIT.PAATOKSET.OTSIKKO.FI}</h1></Typography>
```

Comment:
This should be `<Typography variant="h1">...</Typography>` without manually added h1 element.